rotkehlchen/balances/historical.py

```diff
@@ -111,6 +111,8 @@ def get_asset_amounts(
                 total_amount -= event.balance.amount
 
             amounts[ts_ms_to_sec(event.timestamp)] = total_amount
+            if total_amount < ZERO:
```

Comment:
i don't think this is enough.

The frontend also needs information about the event that caused this.